src/CloudFoundry.CloudController.Test.Integration/PushTest.cs

```diff
@@ -63,7 +63,7 @@ public static void ClassInit(TestContext context)
 
             if (winStack == Guid.Empty)
             {
-                throw new Exception("Could not test on a deployment without a windows 2012 stack");
+                Assert.Inconclusive("Could not test on a deployment without a windows 2012 stack");
```

Comment:
This is just when running the test on a local VS + bosh-lite. The CI has all the categories needed.